Cargo.toml

```diff
@@ -186,17 +187,19 @@ runtime-benchmarks = [
 	"centrifuge-runtime/runtime-benchmarks",
 	"development-runtime/runtime-benchmarks",
 	"runtime-common/runtime-benchmarks",
+
 ]
 test-benchmarks = [
 	"runtime-benchmarks",
 	"runtime-integration-tests/runtime-benchmarks",
+	"pallet-pools/runtime-benchmarks",
```

Comment:
I don't think we need this at this level - it should either be selected manually (by running `cargo test -p pallet-pools --features runtime-benchmarks`), or be transitive via features on the pallets which depend on `pallet-pools` (namely the runtimes)